src/composables/Project.js

```diff
@@ -834,3 +834,19 @@ export async function exists(path) {
     );
   });
 }
+
+/**
+ * Extract project name from url.
+ * @param {String} url - Url to extract project name.
+ * @returns {String|null} Project name.
+ */
+export function extractProjectName(url) {
```

Comment:
Duplicate function, already exists

Reply:
not sure I can use the existing function, since it gets the project name among the existing project. Whereas with `extractProjectName` we need to get the new project name by its url.

I can modify the `getProjectName` function  or keep `extractProjectName` function
```
/**
 * Get name of remote repository or local project if it is not based upon a remote repository.
 * @param {String} projectId - Id of project.
 * @return {String} Project Name.
 */
export function getProjectName(projectId) {
  const projects = getProjects();
  const { repository } = projects[projectId].git;

  if (repository) {
    return repository.split('/').at(-1);
  }

  return projectId;
}
```

Reply:
Keep both, and update getProjectName function like that:
```js
export function getProjectName(projectId) {
  const projects = getProjects();
  const { repository } = projects[projectId].git;

  if (repository) {
    return extractProjectName(repository);
  }

  return projectId;
}
```

Reply:
done